### DISCUSSION TITLE
Add an option to turn a website into a web app

### DISCUSSION THREAD

**Discussion by krafairus** [zen-browser/desktop#3486] [Ideas]
Add an option to turn a website into a web app, like the install apps button available in Chrome but with the great capabilities of Firefox in a well-integrated way.

**Comment by pinage404** [comment]
Duplicate of https://github.com/zen-browser/desktop/issues/560 https://github.com/zen-browser/desktop/issues/379 https://github.com/zen-browser/desktop/discussions/2690 https://github.com/zen-browser/desktop/discussions/800 https://github.com/zen-browser/desktop/discussions/1415
